src/bin/run.rs

```diff
@@ -92,7 +92,10 @@ pub fn execute(options: Options, config: &Config) -> CliResult<Option<()>> {
         None => Ok(None),
         Some(err) => {
             Err(match err.exit.as_ref().and_then(|e| e.code()) {
-                Some(i) => CliError::from_error(err, i),
+                Some(code) => {
+                    let desc = format!("Process finished with exit status {}", code);
+                    CliError::from_boxed(human(desc), code)
```

Comment:
Hm unfortunately this is throwing away `err` which has lots of other contextual information which shouldn't be lost. I'm also a little concerned about displaying this sort of information by default. I see `cargo run` as a "build stuff and then exec" in the sense that the process basically "becomes" the child.